install/kubernetes/cilium/templates/validate.yaml

```
@@ -33,3 +33,13 @@
     {{ fail "ClusterMesh TLS certgen method=certmanager requires user specify .Values.clustermesh.apiserver.tls.auto.certManagerIssuerRef" }}
   {{- end }}  
 {{- end }}
+
+{{/* validate hubble-ui specific config */}}
+{{- if .Values.hubble.ui.enabled }}
+  {{- if .Values.hubble.ui.backend.image.tag | trimPrefix "v" | semverCompare "<0.9.0" }}
```

as the image.tag might contain sha components, should we do the below ? 

```suggestion
  {{- if regexReplaceAll  "@.*$" .Values.hubble.ui.backend.image.tag "" | trimPrefix "v" | semverCompare "<0.9.0" }}
```

---

Yeah, thanks!